rootfs/etc/haproxy/template/haproxy.tmpl

```diff
@@ -127,7 +127,7 @@ frontend httpfront
 ###### HTTPS frontend (tcp mode)
 ######
 frontend httpsfront
-    bind *:443
+    bind *:443{{ if $cfg.UseProxyProtocol }} accept-proxy{{ end }}
```

Comment:
Actually omitting from stats port was intentional because I plan to perhaps use it for readiness and liveness checks, which would break because Kubernetes doesn't send the proxy protocol. What are your thoughts?